product/runtime/src/test/python/chaquopy/test/android/test_import.py

```diff
@@ -581,18 +581,18 @@ def check_top_level(real_name, load_name, id):
         # important enough to investigate just now.
         self.assertFalse(hasattr(imp_rename_2, "mod_3"))
 
-    # Ensure that a package can be imported by a bare name when its in an AssetFinder subdirectory.
+    # Ensure that a package can be imported by a bare name when it's in an AssetFinder subdirectory.
     # This is typically done when vendoring packages and dynamically adjusting sys.path. See #820.
-    def test_imp_subdir(self):
-        sys.modules.pop("about", None)
+    def test_path_subdir(self):
```

Comment:
"imp" in the other tests refers to the `imp` module, which this isn't related to.